src/Tribe/iCal.php

```diff
@@ -84,8 +84,15 @@ public function get_ical_link( $type = 'home' ) {
 	 */
 	public function month_view_ical_link( $event_date = null ) {
 		$tec = Tribe__Events__Main::instance();
+		$url = $tec->getLink( 'month', $event_date );
 
-		return add_query_arg( [ 'ical' => 1 ], $tec->getLink( 'month', $event_date ) );
+		/*
+		 * If the url contains "now" for the date, replace it with the current year-month.
+		 * Make sure we don't replace if it is part of another word - check for slashes.
+		 */
+		$url = str_replace( '/now/', '/' . date( 'Y-m' ) . '/', $url );
```

Comment:
:no_entry_sign: date() is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead.

<sub>hash: 5201e70188e510944596fc849c07fab7</sub>